README.md

```diff
@@ -389,6 +390,52 @@ The secret can be created by executing the following command:
 oc create secret generic keycloak-group-sync --from-literal=username=<username> --from-literal=password=<password>
 ```
 
+### Okta
+
+Groups assigned to Okta applications can be synchronized into OpenShift. The following table describes the set of configuration options for the Okta provider:
```

Comment:
Any links provided by Okta that can be shared with end users?

Reply:
Added some additional links that I think will be help around groups/apps, as we well as docs for the API that `okta.go` calls out to